### PR TITLE
Added check-my-diary-dev vars

### PR DESCRIPTION
**Pull request by sgsollie**

### CODE REVIEW

File: namespace-resources/variables.tf [hunk 1]
```diff
@@ -5,32 +5,42 @@ variable "cluster" {
```
Comment (alkar) [inline namespace-resources/variables.tf]: Please do not commit changes in `namespace-resources`!

Comment (digitalronin) [reply namespace-resources/variables.tf]: To clarify what Mitch said, the stuff you've changed in `namespace-resources/variables.tf` should be in a new file; `namespaces/cloud-platform-live-0.k8s.integration.dsd.io/check-my-diary-dev/resources/variables.tf` (or, you could just fill in the values directly in any `namespaces/cloud-platform-live-0.k8s.integration.dsd.io/check-my-diary-dev/resources/*.tf` files you create.

Comment (sgsollie) [reply namespace-resources/variables.tf]: - yeah sorry I actually wasn't supposed to push that. 

I'll copy that out elsewhere

**Comment by sgsollie** [comment]
Thanks guys - I was just keeping the variables handy for later, that .tf file itself probably isn't really needed to actually proceed.

**Pull request merged**